server/sonar-server/src/test/java/org/sonar/server/usergroups/ws/SearchActionTest.java

```diff
@@ -76,40 +67,49 @@ public void search_empty() throws Exception {
 
   @Test
   public void search_without_parameters() throws Exception {
-    loginAsSimpleUser();
-    insertGroups("users", "admins", "customer1", "customer2", "customer3");
-    dbSession.commit();
+    insertGroup(defaultOrganizationProvider.getDto(), "users", 0);
```

Comment:
since this test never changes the default organization, `defaultOrganizationProvider.getDto()` could be stored in a field of the test class.

it will strongly reduce duplicated code added to the class